code/game/objects/items/cigs_lighters.dm

```diff
@@ -367,6 +367,47 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 
 /obj/item/clothing/mask/cigarette/rollie/Initialize()
 	. = ..()
+	name = pick(list(

```

Comment:
this should be in a string file like other huge lists of texts. joint_names.txt 😏

Reply:
no